1.Exploratory_Data_Analysis/scripts/0.cell_count_analysis.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import sys
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+get_ipython().run_line_magic("matplotlib", "inline")
+import itertools
+import pathlib
+
+import pyarrow.parquet as pq
+from sklearn.cluster import KMeans
+from sklearn.manifold import TSNE
+
+sys.path.append("..")
+from utils.utils import df_stats, plot_features_all_cluster
```

Comment:
A small thing here but the [PEP style guide](https://peps.python.org/pep-0008/#imports) says:

> Imports should be grouped in the following order:
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.

So consider moving the `itertools` and `pathlib` imports up unless I am missing something here.

Reply:
fixed